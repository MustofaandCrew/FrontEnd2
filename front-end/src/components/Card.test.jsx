import React from "react";
import renderer from "react-test-renderer";


describe("Test suits for Card", () => {
    it("should match with snapshot", () => {
        const tree = renderer
            .create(
                    <div className="col h-100">
                        <div className="card h-100">
                            <img src="https://ncrsport.com/img/storage/large/553558-125-1.jpg" className="thumbnail-card" alt="img" data-testid="image" />
                            <div className="card-body">
                                <h5 className="card-title thumbnail-text" data-testid="thumbnail-test">Air Jordan</h5>
                                <p className="card-text price-text" data-testid="price-test">Rp. 5.000.000</p>
                            </div>
                        </div>
                    </div>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});