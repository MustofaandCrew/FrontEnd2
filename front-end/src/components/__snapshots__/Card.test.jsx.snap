// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Card should match with snapshot 1`] = `
<div
  className="col h-100"
>
  <div
    className="card h-100"
  >
    <img
      alt="img"
      className="thumbnail-card"
      data-testid="image"
      src="https://ncrsport.com/img/storage/large/553558-125-1.jpg"
    />
    <div
      className="card-body"
    >
      <h5
        className="card-title thumbnail-text"
        data-testid="thumbnail-test"
      >
        Air Jordan
      </h5>
      <p
        className="card-text price-text"
        data-testid="price-test"
      >
        Rp. 5.000.000
      </p>
    </div>
  </div>
</div>
`;
